tox.ini

```diff
@@ -37,4 +37,4 @@ exclude =
     alembic,
     .circleci
 max-complexity = 10
-max-line-length = 80
+max-line-length = 80
```

Comment:
Add an end of line, otherwise nice work

Reply:
Thanks for the feedback, I have implemented the feedback